src/test/scala/sample.scala

```diff
@@ -2,14 +2,13 @@ package sample
 
 import nequi.zio.logger._
 
-import scalaz._
-import scalaz.zio._
+import scalaz.Scalaz._
 
-import Scalaz._
+import zio._
 
 object SampleApp extends App {
   def run(args: List[String]): ZIO[Environment, Nothing, Int] = {
-    val logged: ZIO[Logger, Nothing, Unit] = info("Hello World")
+    val logged: ZIO[Logger, Nothing, Unit] = info(42)
```

Comment:
I'm afraid this will not compile...
What about sth. like this:  
```val logged: ZIO[Logger, Nothing, Unit] = info(s"Logged some value: ${44}")```

Reply:
Additionally to get this example to log anything you need to add some logger implementation dependency.
Maybe you could add following to build.sbt:

```libraryDependencies ++= Seq("org.slf4j" % "slf4j-simple" % "1.7.5" %Test)```

Reply:
It compiles because the Scalaz dependency is on the classpath and the implicit conversion will therefore be available.